translator/tocwconfig/sampleConfig/advanced_config_darwin.yaml

```diff
@@ -10,13 +10,11 @@ exporters:
             enabled: true
 extensions:
     agenthealth/metrics:
+        is_status_code_enabled: true
         is_usage_data_enabled: true
         stats:
             operations:
-                - PutMetricData
-            usage_flags:
-                mode: EC2
-                region_type: ACJ
+                - '*'
```

Comment:
This extension configuration is wrong and should not have changed. Please review the YAML files that are being generated.